CHANGES.md

```diff
@@ -10,6 +10,8 @@
 
 ### Removed
 
+- Remove unused `Package_summary.equal` and `Package_summary.pp` functions (#356, @gridbugs)
```

Comment:
This doesn't need a changelog entry I would say since it's not user-visible (at least if the user doesn't use `Duniverse_lib` but I'd argue that the user facing side of opam-monorepo is the binary, not the library).

I've tagged it as "no changelog" so the CI will pass without requiring an entry.